### PR TITLE
Add a DEV section to CHANGELOG

### PR DESCRIPTION
**Pull request by blaix**
How do you feel about adding a section to track ongoing changes that haven't been released yet?

I did this because #26 had some commits that were actually worthy of being included in the CHANGELOG, but feared they may be missed when the next engineer does a release, looks at the merged pull and sees it was just "integration tests".

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by thomasw** [comment]
Sounds good. :+1:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,14 @@
+DEV
+----
+
+Improvements:
+
+* return created subscription from Yola.create_subscription (5f7aba284e4678fa10bcc6482c8f04168acc831c)
```
Comment (stefanor) [inline CHANGELOG.rst:6]: I think linking to commits / PRs can compromise the readability of a changelog, without significant benefit.

It should be easy enough to find where something was implemented, if one of:
- releases are small and frequent
- the changelog entry is added together with the feature (in a DEV section, for example)
- commits are well described

Comment (blaix) [inline CHANGELOG.rst:6]: > the changelog entry is added together with the feature (in a DEV section, for example)

Yeah. I would prefer that to linking to the commit. That's not what happened here, but I don't want to set this precedent. I'll remove the commit links.

**Pull request closed without merging**